File: day3/Test5.java
package com.day3;

public class Test5 {

	public static void main(String[] args) {
		
		int i, j;
		
		
		// 정삼각형 만들기 
		for(i=1; i<=5; i++) {
			
			for(j=1; j<=5-i; j++) {
				System.out.print(" ");
			}
			
			for(j=1; j<=i*2-1; j++) {
				System.out.print("*");
			}
			System.out.println();
		}
		System.out.println();
		//역삼각형
		for(i=5; i>=1; i--) {
			
			for(j=1; j<=5-i; j++) {
				System.out.print(" ");
			}
			
			for(j=1; j<=i*2-1; j++) {
				System.out.print("*");
			}
			System.out.println();
		}
		
	}
}

File: test6/src/components/Test5.js
import React from 'react';
import img0 from '../s_images/cat0.jpg'
import img1 from '../s_images/cat1.jpg'



const Test5 = () => {
    return (
        <div>
            <img src={img0} alt=''/>
            <img src={img1} alt=''/>
            {/* p_images에있는 그림은 import가 안되서 이렇게 */}
            <hr/> 
            <img src='./p_images/cat3.jpg' alt=''/>
        </div>
        
    );
};

export default Test5;